### PR TITLE
Feature/create flower

### PR DESCRIPTION
**Pull request by dodgecu**

### CODE REVIEW

File: assets/styles/base/_base.scss [hunk 1]
```diff
@@ -0,0 +1,16 @@
+*,
+*::after,
+*::before {
+  margin: 0;
+  padding: 0;
+  box-sizing: inherit;
+}
+
+html {
```
Comment (enheit) [inline assets/styles/base/_base.scss]: don't assign to the html & body tag

File: src/CreateFlowerPage/CreateFlowerForm/CreateFlowerForm.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import { reduxForm } from "redux-form";
```
Comment (enheit) [inline src/CreateFlowerPage/CreateFlowerForm/CreateFlowerForm.js]: Add space between imports

File: src/CreateFlowerPage/CreateFlowerForm/CreateFlowerForm.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+import ValidateFields from "./ValidateFields";
+import Name from "./Fields/Name";
+import FlowerType from "./Fields/FlowerType";
+import AirHumidity from "./Fields/AirHumidity";
```
Comment (enheit) [inline src/CreateFlowerPage/CreateFlowerForm/CreateFlowerForm.js]: Use barrel file

File: src/CreateFlowerPage/CreateFlowerForm/CreateFlowerForm.js [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+          type="submit"
+          className="create-flower__submit"
+          value="Add FLower"
+        />
+      </form>
+    </div>
+  );
+};
+
+export default reduxForm({
```
Comment (enheit) [inline src/CreateFlowerPage/CreateFlowerForm/CreateFlowerForm.js]: Consider to use `compose` function from `redux`. This is not necessary in the component. FYI.

File: src/CreateFlowerPage/CreateFlowerForm/Fields/FlowerType.js [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+class flowerType extends Component {
+  render() {
+    return (
+      <Field
+        id="flower-select"
+        name="flowerSelect"
+        label="Flower Type"
+        component={this.props.validForm}
+      >
+        {flowers.map(flower => {
```
Comment (enheit) [inline src/CreateFlowerPage/CreateFlowerForm/Fields/FlowerType.js]: Move the logic to the separate function. For example: `flowers.map(flower => this.renderOption(flower))`

File: src/CreateFlowerPage/CreateFlowerForm/ValidateFields.js [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+    </label>
+    {render(input, label, id, name, rest)}
+
+    {meta.error && meta.touched && (
+      <span className="form__group--error">{meta.error}</span>
+    )}
+  </div>
+);
+
+const validate = field => {
```
Comment (enheit) [inline src/CreateFlowerPage/CreateFlowerForm/ValidateFields.js]: Consider to move it in the separate file.

File: src/test.js [hunk 7]
```diff
@@ -0,0 +1,7 @@
+import React from "react";
```
Comment (enheit) [inline src/test.js]: Do you really need this file? If not, remove.

File: assets/styles/abstracts/_variables.scss [hunk 8]
```diff
@@ -0,0 +1,3 @@
+$primary: #abb3cd;
```
Comment (m-ost) [inline assets/styles/abstracts/_variables.scss]: change a primary color to #15D94A

**Pull request merged**